src/main/java/com/gregtechceu/gtceu/integration/ae2/machine/MEPatternBufferPartMachine.java

```diff
@@ -415,6 +419,17 @@ public void onMachineRemoved() {
         clearInventory(shareInventory);
     }
 
+    @Override
```

Comment:
Nitpick-ish, but you could probably make these methods default and return `InteractionResult.PASS` in the interface, and then just not override them if you don't want anything to happen. Same with the proxy shift use.

Reply:
fair